demo-python/code/azure-search-vector-image-python-sample.ipynb

```diff
@@ -238,14 +236,14 @@
     "index_client = SearchIndexClient(\n",
     "    endpoint=service_endpoint, credential=credential)\n",
     "fields = [\n",
-    "    SimpleField(name=\"id\", type=SearchFieldDataType.String, key=True, sortable=True),  \n",
+    "    SimpleField(name=\"id\", type=SearchFieldDataType.String, key=True, sortable=True, filterable=True, facetable=True),  \n",
```

Comment:
is this intentional change for filterable/facetable true on ID field?

Reply:
Yes, having an ID field as sortable and filterable helps with potential backup and restore scenarios.